src/http_rest_api_handler.cpp

```diff
@@ -499,7 +499,7 @@ Status HttpRestApiHandler::processV3(const std::string_view uri, const HttpReque
         serverReaderWriter->OverwriteResponseHeader("Connection", "keep-alive");
         auto status = executor->inferStream(request, *serverReaderWriter);
         if (!status.ok()) {
-            sendErrorImpl(status.string(), *serverReaderWriter);
+            sendErrorImplWithCode(status.string(), 400, *serverReaderWriter);
```

Comment:
I dont think we need to put sendErrorImplWithCode in http_graph_executor_impl.hpp. This file is supposed to contain implementations for templated methods used in graph executor (http and grpc). I would move the implementation here

Reply:
+ use enum instead of 400